guides/common/modules/proc_deleting_a_task_by_id.adoc

```diff
@@ -0,0 +1,32 @@
+[id="Deleting_a_Task_by_ID_{context}"]
+= Deleting a Task by ID
+
+You can delete tasks by ID, for example if you have submitted confidential data by mistake.
+
+.Procedure
+. Connect to your {ProjectServer} using SSH:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# ssh root@{foreman-example-com}
+----
+. Optional: View the task:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# hammer task info --id _My_Task_ID_
+----
+. Delete the task:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# foreman-rake foreman_tasks:cleanup TASK_SEARCH="id=_My_Task_ID_"
+----
+. Optional: Ensure the task has been removed from {ProjectServer}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# hammer task info --id _My_Task_ID_
+----
++
+Note that this command returns a non-zero exit code.
```

Comment:
What does this mean to the user?

Reply:
So potentially, your terminal emulator aka shell bleeps or marks your command prompt red if the previous command does not have zero as return code:

<img width="327" alt="Screenshot 2022-05-30 at 17 01 27" src="https://user-images.githubusercontent.com/12595287/171019029-8cd1d458-1a61-4b9b-9fe5-b68b6f88cc5c.png">

false returns 1, true returns zero, `$?` is the return code of the previously executed command.

By definition, anything non-zero means it failed. Users should always stop their procedure and investigate if a command returns anything else than zero.

Reply:
```suggestion
Note that because the task is deleted, this command returns a non-zero exit code.
```
Can we change to something like this?

Reply:
Applied.